src/Compilers/CSharp/Portable/Symbols/Metadata/PE/PEParameterSymbol.cs

```diff
@@ -882,7 +882,9 @@ private ImmutableArray<int> DecodeInterpolatedStringHandlerArgumentAttribute()
                 switch (name)
                 {
                     case null:
-                    case "" when !ContainingSymbol.RequiresInstanceReceiver() || ContainingSymbol is MethodSymbol { MethodKind: MethodKind.Constructor or MethodKind.DelegateInvoke }:
+                    case "" when !ContainingSymbol.RequiresInstanceReceiver()
+                                 || ContainingSymbol is MethodSymbol { MethodKind: MethodKind.Constructor or MethodKind.DelegateInvoke }
+                                                     or MethodSymbol { ContainingSymbol: TypeSymbol { IsExtension: true } }:
```

Comment:
>or MethodSymbol { ContainingSymbol: TypeSymbol { IsExtension: true } } [](http://example.com/codeflow?start=53&length=70)

In the long term, I think it would be better to use condition of the form like ```ContainingSymbol.GetIsNewExtensionMembere()```. It should work for indexers as well, etc.